app/adapters/review-action.ts

```diff
@@ -1,7 +1,7 @@
 import OsfAdapter from './osf-adapter';
 
 export default class ReviewActionAdapter extends OsfAdapter {
-    parentRelationship = 'provider';
+    parentRelationship = 'target';
```

Comment:
`parentRelationship` is changed to `target` because we post to `/registrations/<registratioId>/actions` endpoint to create a review action.